testing/python/metafunc.py

```diff
@@ -448,7 +448,10 @@ def test_idmaker_enum(self) -> None:
         enum = pytest.importorskip("enum")
         e = enum.Enum("Foo", "one, two")
         result = idmaker(("a", "b"), [pytest.param(e.one, e.two)])
-        assert result == ["Foo.one-Foo.two"]
+        assert result in [
```

Comment:
I suggest to break this using an if:

```python
if sys.version_info[:2] >= (3, 10):
    assert result == ["one-two"]
else:
    assert result == ["Foo.one-Foo.two"]
```

Having explicit `version_info` checks makes it much easier to find dead code once we drop Python versions. 👍